api/types/client.go

```diff
@@ -126,21 +126,21 @@ type ImageBuildOptions struct {
 	Remove         bool
 	ForceRemove    bool
 	PullParent     bool
-	Isolation      string
-	CPUSetCPUs     string
-	CPUSetMems     string
-	CPUShares      int64
-	CPUQuota       int64
-	CPUPeriod      int64
-	Memory         int64
-	MemorySwap     int64
-	CgroupParent   string
-	ShmSize        string
-	Dockerfile     string
-	Ulimits        []*units.Ulimit
-	BuildArgs      []string
-	AuthConfigs    map[string]AuthConfig
-	Context        io.Reader
+	container.IsolationLevel
```

Comment:
should this be `Isolation container.IsolationLevel` I think you lost the variable name. It still works because the struct is embedded and you can reference it with `options.IsolationLevel`, but I'm not sure if it's expected or not.